cmd/pool/sidecar.go

```diff
@@ -110,9 +111,16 @@ func sidecarOffer(ctx *cli.Context) error {
 			args = args.Tail()
 		}
 
+		// var err error
+		trader_key, err := parseAccountKey(ctx, args)
```

Comment:
We should add the `acct_key` to the end of the list of arguments in `ArgsUsage` above.